lib/cc/analyzer/container_listener.rb

```diff
@@ -0,0 +1,22 @@
+module CC
+  module Analyzer
+    class ContainerListener
+      ContainerData = Struct.new(
```

Comment:
What's the benefit of nesting this under `ContainerListener` instead of `Container`? Nothing in this class actually uses the struct directly. You could even do this `Struct.new` call right within `container_data` above